### ISSUE TITLE
Entering a latex equation within a node, removes newlines (<br>) from the rendered output

### ISSUE THREAD

**Issue by npapnet** [mermaid-js/mermaid#5941]
### Description

When I start a graph in mermaid live 

```
graph TB
    A["This should be <br> but its not "]--"$$\int A dx$$"--> B
``` 
The new line is properly rendered. 

However when I add a latex environment $$ ... $$ like the following, the new line disappers and all the text is rendered in one line. 

```
graph TB
    A["This should be <br> but its not $$\int_a^b$$"]--"$$\int A dx$$"--> B
```

I am pretty positive that this was not the case before. 

### Steps to reproduce

Just add the code above in mermaid live

### Screenshots

New line rendered
![image](https://github.com/user-attachments/assets/5569ba31-9be1-4a00-8987-b92d7fde0b8e)

Text not rendered 
![image](https://github.com/user-attachments/assets/d91019bb-3f22-40ca-ad09-7c1da64dcaee)

### Setup

- Mermaid version: 11.3.0 Live editor
- Browser and Version: Edge, and Firefox

**Comment by Ceredril** [comment]
Did a bit of testing:

Breaking version for the npm package is (@mermaid-js/mermaid-cli@11.1.1)
The previous version (@mermaid-js/mermaid-cli@10.9.1) still formats latex and line breaks together fine.

**Comment by yunhengchen** [comment]
Would like to bump this issue again !  Had troubles in designing an algorithm flowchart when I want to break line the lines for Input and Output using mathematical symbols. Any further help would be greatly appreciated.

**Comment by philousoph** [comment]
Yes, still a bug in the newest 11.11.0, which is quite annoying. 
I am using Quarto so I need a workaround for now (Thanks to @Ceredril for pointing me to the working version):
```
format:
    include-in-header:
      text: |
       <script src="https://cdn.jsdelivr.net/npm/mermaid@10.9.1/dist/mermaid.min.js"></script>
```
